packages/client/src/generation/TSClient/Count.ts

```diff
@@ -84,7 +84,7 @@ class CountDelegate implements Generatable {
     if (!mapping) {
       return ''
     }
-    const model = this.dmmf.modelMap[name]
+    const model = this.dmmf.typeModelMap[name]
```

Comment:
Maybe could be better if we also rename the const? It can be confusing now I think.
From `const model` to `const modelType` maybe?